File: app/src/main/java/com/musicplayer/SocyMusic/ui/album/AlbumSongHolder.java
package com.musicplayer.SocyMusic.ui.album;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.musicplayer.SocyMusic.MediaPlayerUtil;
import com.musicplayer.SocyMusic.data.Song;
import com.musicplayer.musicplayer.R;

public class AlbumSongHolder extends RecyclerView.ViewHolder {
    private TextView songTitleTextView;
    private TextView songLengthTextView;

    public AlbumSongHolder(@NonNull View itemView, AlbumSongAdapter.ItemClickListener clickListener) {
        super(itemView);
        itemView.setOnClickListener(view -> clickListener.onItemClick(getBindingAdapterPosition(), view));
        itemView.setOnLongClickListener(view -> clickListener.onItemLongClick(getBindingAdapterPosition(), view));
        songTitleTextView = itemView.findViewById(R.id.textview_album_song_item_title);
        songLengthTextView = itemView.findViewById(R.id.textview_album_song_item_length);
    }

    public void bind(Song song) {
        songTitleTextView.setText(song.getTitle());
        songLengthTextView.setText(MediaPlayerUtil.createTime(song.getDuration()));
    }
}

File: app/src/main/java/com/musicplayer/OpenMusic/ui/album/AlbumSongHolder.kt
package com.musicplayer.OpenMusic.ui.album

import android.view.View
import com.musicplayer.OpenMusic.MediaPlayerUtil.createTime
import androidx.recyclerview.widget.RecyclerView
import android.widget.TextView
import com.musicplayer.musicplayer.R
import com.musicplayer.OpenMusic.data.Song

class AlbumSongHolder(itemView: View, clickListener: AlbumSongAdapter.ItemClickListener) :
    RecyclerView.ViewHolder(itemView) {
    private val songTitleTextView: TextView
    private val songLengthTextView: TextView

    init {
        itemView.setOnClickListener { view: View ->
            clickListener.onItemClick(
                bindingAdapterPosition, view
            )
        }
        itemView.setOnLongClickListener { view: View ->
            clickListener.onItemLongClick(
                bindingAdapterPosition, view
            )
        }
        songTitleTextView = itemView.findViewById(R.id.textview_album_song_item_title)
        songLengthTextView = itemView.findViewById(R.id.textview_album_song_item_length)
    }

    fun bind(song: Song) {
        songTitleTextView.text = song.title
        songLengthTextView.text = createTime(song.extractDuration())
    }
}